### PR TITLE
5.0-to-5-take-2

### PR DESCRIPTION
**Pull request by recrwplay**

### CODE REVIEW

File: common-content/modules/ROOT/partials/cypher-workflow.adoc [hunk 1]
```diff
@@ -647,7 +647,7 @@ Inbound conversion is carried out using http://cldr.unicode.org/development/deve
 # tag::exceptions-errors[]
 
 When executing *Cypher* or carrying out other operations with the driver, certain exceptions and error cases may arise.
-*_Server-generated exceptions_* are each associated with a link:/docs/status-codes/5.0#status-codes[*_status code_*] that describes the nature of the problem and a message that provides more detail.
+*_Server-generated exceptions_* are each associated with a link:{neo4j-docs-base-uri}/status-codes/5.0#status-codes[*_status code_*] that describes the nature of the problem and a message that provides more detail.
```
Comment (stefano-ottolenghi) [inline common-content/modules/ROOT/partials/cypher-workflow.adoc]: Should this `5.0` also be changed?

File: common-content/modules/ROOT/partials/get-started.adoc [hunk 2]
```diff
@@ -59,7 +59,7 @@ In the general case, only the xref:client-applications.adoc#driver-connection-ur
 The official drivers do not support HTTP communication.
 If you need an HTTP driver, choose one of the community drivers.
 
-See also the link:/docs/http-api/5.0#http-api[HTTP API documentation].
+See also the link:{neo4j-docs-base-uri}/http-api/5.0#http-api[HTTP API documentation].
```
Comment (stefano-ottolenghi) [inline common-content/modules/ROOT/partials/get-started.adoc]: Should this `5.0` also be changed?

File: common-content/modules/ROOT/partials/cypher-workflow.adoc (hunk 1 above)
Comment (recrwplay) [reply common-content/modules/ROOT/partials/cypher-workflow.adoc]: Yes - well spotted - thanks!

File: common-content/modules/ROOT/partials/get-started.adoc (hunk 2 above)
Comment (recrwplay) [reply common-content/modules/ROOT/partials/get-started.adoc]: This one too - thanks!

**Comment by stefano-ottolenghi** [comment]
LGTM :)

**Review by stefano-ottolenghi — APPROVED** [review]

**Pull request merged**